store/mongo/datastore_mongo.go

```diff
@@ -2774,3 +2774,25 @@ func (db *DataStoreMongo) SaveLastDeviceDeploymentStatus(
 	}
 	return err
 }
+
+func (db *DataStoreMongo) GetLastDeviceDeploymentStatus(
+	ctx context.Context,
+	devicesIds []string,
+) ([]model.DeviceDeploymentLastStatus, error) {
+	database := db.client.Database(DatabaseName)
```

Comment:
in all other places we get the db name from context, which will result in the same name as you have here, but will look different in the enterprise, so I'm guessing there will be some conflicts?
sorry for the messy comment

Reply:
I assume we will migrate to single db one day, so thats why I am keeping it like that. do you want me to get the db from context?

Reply:
yes, like we do in the other places; this needs to work before migration too, and, despite the fact that we never use OS with multi tenancy, in theory, it's possible to use it this way